### PR TITLE
Add transpiler plugins section to web page

### PR DESCRIPTION
**Pull request by frankharkins**
### Summary

This PR adds a transpiler plugins section to the web page.

### CODE REVIEW

**Review by Eric-Arellano — COMMENTED** [review]
I think providers should probably remain on top, right? Note that we used to have a dedicated page for providers—vs being in ecosystem—because it was important enough.

File: ecosystem/html_templates/webpage.html.jinja [hunk 1]
```diff
@@ -75,6 +75,38 @@
...
+              d="M4.19,24.82a4,4,0,0,1,0-5.67l8-8a4,4,0,0,1,5.67,0A3.94,3.94,0,0,1,19,14a4,4,0,0,1-1.17,2.85L15.71,19l1.42,1.42,2.12-2.12a6,6,0,0,0-8.51-8.51l-8,8a6,6,0,0,0,0,8.51A6,6,0,0,0,7,28a6.07,6.07,0,0,0,4.28-1.76L9.86,24.82A4,4,0,0,1,4.19,24.82Z"
+            ></path>
+          </svg>
+        </a>
+        Transpiler plugins
+      </h2>
+      <p>
+        Use these projects to compile and optimize your quantum circuits.
+      </p>
+      <div id="section1" class="cards cds--row">{{section_transpiler_plugin_cards}}</div>
```
Comment (arnaucasau) [inline ecosystem/html_templates/webpage.html.jinja:108]: Little comment: This id seems to be duplicated, but I think we don't need the id anymore in any section div and I forgot to remove it before. Could you remove it from the 4 sections, please?

**Review by arnaucasau — APPROVED** [review]
Looks good to me. Thanks for adding the section Frank!

File: ecosystem/html_templates/webpage.html.jinja (hunk 1 above)
Comment (frankharkins) [reply ecosystem/html_templates/webpage.html.jinja]: Great spot, thanks!

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**